### ISSUE TITLE
Should built-in states be reflected in the API

### ISSUE THREAD

**Issue by plinss** [WICG/custom-state-pseudo-class#10]
Given the change in syntax, and that all custom states now have to be prefixed with '--', it raises the question of whether built-in states should also be reflected in the API.

e.g. if an element is being hovered over, should `el.states.has('hover')` be true? 

This fits with Houdini's mission to explain existing CSS behavior.

Of course, the user should no be able to add or clear built-in state.

**Comment by tkent-google** [comment]
Yeah, we can extend the feature for existing built-in pseudo classes.

https://github.com/w3c/webcomponents/issues/813
